machineSetup/setupPerfMachine.ps1

```diff
@@ -0,0 +1,144 @@
+﻿#Set power scheme to high performance
+powercfg /S 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c
```

Comment:
You might require, at the top of the script, running as administrator by checking something like this:
`([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")`